saleor/warehouse/models.py

```diff
@@ -106,6 +106,14 @@ def _for_country_lines_and_stocks(
             )
         )
 
+    def _for_country_click_and_collect(self, country: str) -> QuerySet["Warehouse"]:
+        warehouse_cc_option_enum = WarehouseClickAndCollectOption
+
```

Comment:
What is the reason for assigning `WarehouseClickAndCollectOption` to variable instead of inline access to the values?